stan-ballerina/tests/certs/ca.crt

```diff
@@ -0,0 +1,13 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
These certs will expire at July 29, 2021. Shall we create certs valid at least 10 years since they are used for tests?

Reply:
I don't think this library will last 10 years but will do update 😀

Reply:
Yeah. But definitely will last more than 1 month 😄

Reply:
Updated in 765efe8314cbca216101908e0442597778dfef5a